webapp/src/components/backstage/playbook_list.tsx

```diff
@@ -117,13 +117,13 @@ const PlaybookList = () => {
                 key={p.id}
                 onClick={() => view(p)}
             >
-                <a className='col-sm-4 title'>
+                <div className='col-sm-4 title'>
                     <TextWithTooltip
                         id={p.title}
                         text={p.title}
                     />
-                    <TeamNameWithPadding>{teams.length > 1 ? ' (' + getTeamName(teams, p.team_id) + ')' : ''}</TeamNameWithPadding>
-                </a>
+                    {teams.length > 1 ? <TeamName>{getTeamName(teams, p.team_id)}</TeamName> : '' }
```

Comment:
For this case, I would suggest the following pattern:
```suggestion
                    {teams.length > 1 && <TeamName>{getTeamName(teams, p.team_id)}</TeamName>}
```

effectively rendering `null` vs. the empty string altogether.

Reply:
nice!